logdetective/prompts.yml

```diff
@@ -0,0 +1,50 @@
+prompt_template: |
```

Comment:
since we already use jinja2, how about using it for templating here as well?

Reply:
It's a template, but not a jinja template. We are using `format` function for substitution. Using jinja template here would mean splitting `prompts.yml` into multiple files, then passing them to a rendering engine, and finally using resultant strings.

When we only need to deal with one variable, that seems like an overkill.

Reply:
okay, that's fair; we can always adapt jinja later if we wanted to